server/internal/graphql/resolver/pattern.go

```diff
@@ -21,8 +21,8 @@ func (r *Resolver) fetchPatterns(ctx context.Context, provider models.Provider,
 		return nil, err
 	}
 
-	mc := handlers.NewContentModifier(tokenString, provider, prefObj, user.UserID)
-	err = mc.AddMetadataForPatterns(ctx, &resp)
+	// mc := handlers.NewContentModifier(tokenString, provider, prefObj, user.UserID)
```

Comment:
It's not obvious that this code invokes a dry-run. Will you please add comments, describing this?

Reply:
This func `AddMetadataForPatterns` process each pattern file fetched, https://github.com/meshery/meshery/blob/a190c7ecb88ac8fd32e2d2fd6f66dddb6b9c57a7/server/handlers/content_modifier.go#L53

and uses pattern engine to validate it (dry run) 
https://github.com/meshery/meshery/blob/a190c7ecb88ac8fd32e2d2fd6f66dddb6b9c57a7/server/handlers/content_modifier.go#L81

and finally add the results in the patterns fetched from the provider.

Reply:
Thank you, @MUzairS15. Will you please add these comments in the code, too?